xknxproject/models/models.py

```diff
@@ -47,6 +54,27 @@ def __repr__(self) -> str:
         )
 
 
+@dataclass
+class XMLGroupRange:
+    """Class that represents a group range."""
+
+    name: str
+    range_start: int
+    range_end: int
+    group_addresses: list[int]
+    children: list[XMLGroupRange]
+    comment: str
+
+    def str_address(self) -> str:
+        """Generate a string representation for the range (same as in in ETS)."""
```

Comment:
```suggestion
        """Generate a string representation for the range."""
```
it isn't since we always use 3-level here 😉